patches/tModLoader/Terraria/ID/ContainerID.cs

```diff
@@ -0,0 +1,83 @@
+namespace Terraria.ID;
+
+/// <summary>
+/// The different IDs for the subtypes inside TileID.Containers.
+/// </summary>
+public static class ContainerID
+{
+	public const int Chest = 0;
+	public const int GoldenChest = 1;
+	public const int LockedGoldenChest = 2;
+	public const int ShadowChest = 3;
+	public const int LockedShadowChest = 4;
+	public const int Barrel = 5;
+	public const int TrashCan = 6;
+	public const int EbonWoodChest = 7;
+	public const int RichMahoganyChest = 8;
+	public const int PearlwoodChest = 9;
+	public const int IvyChest = 10;
+	public const int FrozenChest = 11;
+	public const int LivingWoodChest = 12;
+	public const int SkywareChest = 13;
+	public const int ShadewoodChest = 14;
+	public const int WebCoveredChest = 15;
+	public const int LihzahrdChest = 16;
+	public const int WaterChest = 17;
+	public const int JungleChest = 18;
+	public const int CorruptionChest = 19;
+	public const int CrimsonChest = 20;
+	public const int HallowedChest = 21;
+	public const int IceChest = 22;
+	public const int LockedJungleChest = 23;
+	public const int LockedCorruptChest = 24;
+	public const int LockedCrimsonChest = 25;
+	public const int LockedHallowedChest = 26;
+	public const int LockedIceChest = 27;
+	public const int DynastyChest = 28;
+	public const int HoneyChest = 29;
+	public const int SteampunkChest = 30;
+	public const int PalmWoodChest = 31;
+	public const int MushroomChest = 32;
+	public const int BorealWoodChest = 33;
+	public const int SlimeChest = 34;
+	public const int GreenDungeonChest = 35;
+	public const int LockedGreenDungeonChest = 36;
+	public const int PinkDungeonChest = 37;
+	public const int LockedPinkDungeonChest = 38;
+	public const int BlueDungeonChest = 39;
+	public const int LockedBlueDungeonChest = 40;
+	public const int BoneChest = 41;
+	public const int CactusChest = 42;
+	public const int FleshChest = 43;
+	public const int ObsidianChest = 44;
+	public const int PumpkinChest = 45;
+	public const int SpookyChest = 46;
+	public const int GlassChest = 47;
+	public const int MartianChest = 48;
+	public const int MeteoriteChest = 49;
+	public const int GraniteChest = 50;
+	public const int MarbleChest = 51;
+	public const int CrystalChest = 52;
+	public const int GoldenChest = 53;
+}
+
+/// <summary>
+/// The different IDs for the subtypes inside TileID.Containers2.
+/// </summary>
+public static class ContainerID2
+{
+	public const int Fake_newchest1 = 0; // *
+	public const int Fake_newChest2 = 1; // These two are unused in vanilla
+	public const int SpiderChest = 2;
+	public const int LesionChest = 3;
+	public const int DeadMansChest = 4;
+	public const int SolarChest = 5;
+	public const int VortexChest = 6;
+	public const int NebulaChest = 7;
+	public const int StardustChest = 8;
+	public const int GolfChest = 9;
+	public const int SandstoneChest = 10;
+	public const int BambooChest = 11;
+	public const int DesertChest = 12;
```

Comment:
This won't compile. Typo?

Reply:
Forgot to add Locked in front of it. I guess I didn't open the project correctly since the IDE didn't complain. Also, would it be better to multiply the values by 36 (chest width) in advance in the ID classes?

Reply:
No, these represent the styles, not the coordinates of the top left tile.